docusaurus/docs/iOS/03-guides/01-client-auth.mdx

```diff
@@ -60,6 +60,27 @@ let streamVideo = StreamVideo(
 )
 ```
 
+Anonymous users don't establish an active web socket connection, therefore they won't receive any events. They are just able to watch a livestream or join a call.
+
+The token for an anonymous user should contain the `call_cids` field, which is an array of the call `cid`'s that the user is allowed to join.
```

Comment:
Is there a reference we point to regarding what is a cid?

Reply:
Added more details to explain this.